src/modules/fancyzones/editor/FancyZonesEditor/CanvasEditorWindow.xaml

```diff
@@ -1,4 +1,5 @@
 ﻿<local:EditorWindow x:Class="FancyZonesEditor.CanvasEditorWindow"
+        AutomationProperties.Name="CanvasEditorWindow"
```

Comment:
I feel like this should be inside resource files like
https://github.com/microsoft/PowerToys/blob/master/src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw#L200

Reply:
Done.